stubs/pluggy/pluggy/_result.pyi

```diff
@@ -0,0 +1,21 @@
+from _typeshed import ExcInfo
+from collections.abc import Callable
+from typing import Generic, TypeVar
+
+_T = TypeVar("_T")
+
+class HookCallError(Exception): ...
+
+class _Result(Generic[_T]):
+    _result: _T
+    _exception: BaseException
```

Comment:
Hmm, do we need these private attributes? (Same point that I mentioned earlier about typeshed usually only providing stubs for public API)

```suggestion
```